src/domain/entities/figma-team.ts

```diff
@@ -0,0 +1,16 @@
+export enum FigmaTeamAuthStatus {
+	OK = 'OK',
+	ERROR = 'ERROR',
+}
+
+export type FigmaTeam = {
+	readonly id: number;
+	readonly webhookId: string;
+	readonly teamId: string;
+	readonly teamName: string;
+	readonly figmaAdminAtlassianUserId: string;
+	readonly status: FigmaTeamAuthStatus;
```

Comment:
**question:** Justin, do we need to rename this to `authStatus`?

Reply:
Agreed that it's a bit clearer, have renamed this column and updated the relevant methods/domain types.